orchestra/static/orchestra/project/js/controllers.js

```diff
@@ -0,0 +1,870 @@
+(function () {
+  // 'use strict';
+
+  angular
+    .module('orchestra.project.controllers')
+    .controller('ProjectController', ProjectController);
+
+  ProjectController.$inject = ['$location', '$scope', '$routeParams', '$http', '$sce',
+                            '$compile', '$modal', '$timeout', 'dataService', 'orchestraApi'];
+
+  function ProjectController($location, $scope, $routeParams, $http, $sce,
+                          $compile, $modal, $timeout, dataService, orchestraApi) {
+    // var vm = this;
+    vm = $scope;
+    $scope.activate = function() {
+      // Prevent initial animation of vis load
+      d3.select('.project-management').style('opacity', 0);
+      $scope.vis = new projectVis($routeParams.projectId, '.project-management .svg-wrapper');
+      $scope.vis.update();
+      d3.select('.project-management').transition()
+        .delay(500)
+        .style('opacity', 1);
+    };
+    $scope.activate();
+
+    function projectVis(projectId, parentSelector) {
```

Comment:
Throw all of the *Vis functions inside their own .js file.  if you want, you can go so far as to make one file per *Vis function, but I don't think you need to go that far.

This way, you'll separate the d3 stuff from the angular stuff.